client/me/purchases/manage-purchase/payment-info-block.tsx

```diff
@@ -36,7 +36,18 @@ export default function PaymentInfoBlock( {
 	}
 
 	if ( hasPaymentMethod( purchase ) && isPaidWithCredits( purchase ) ) {
-		return <PaymentInfoBlockWrapper>{ translate( 'Credits' ) }</PaymentInfoBlockWrapper>;
+		return (
+			<PaymentInfoBlockWrapper>
+				<div className="manage-purchase__no-payment-method">
+					<Icon icon={ warning } />
+					{ translate( 'You don’t have a payment method to renew this subscription' ) }
+				</div>
+			</PaymentInfoBlockWrapper>
+		);
+	}
+
+	if ( ! purchase.isAutoRenewEnabled && isPaidWithCredits( purchase ) ) {
+		return <PaymentInfoBlockWrapper>{ translate( 'None ' ) }</PaymentInfoBlockWrapper>;
```

Comment:
It's hard to see, but you've added a space after the word `'None '` which will probably make this be re-translated. I suspect that was a typo.